internal/services/synapse/synapse_workspace_resource.go

```diff
@@ -430,7 +430,7 @@ func resourceSynapseWorkspaceCreate(d *pluginsdk.ResourceData, meta interface{})
 
 	sqlAdmin := expandArmWorkspaceAadAdmin(d.Get("sql_aad_admin").([]interface{}))
```

Comment:
It would probably need an `expandArmWorkspaceAadAdmin` function to expand here properly